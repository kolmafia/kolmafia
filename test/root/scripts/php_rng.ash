rng r = php_seed(1);
rng s = php_seed(2147483647);
print(php_mt_rand(r));
print(php_mt_rand(s));
print(php_mt_rand(r));
print(php_mt_rand(s));

r = php_seed(1);
s = php_seed(2147483647);
print(php_rand(r));
print(php_rand(s));
print(php_rand(r));
print(php_rand(s));