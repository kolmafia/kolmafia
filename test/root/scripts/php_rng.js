const { phpSeed, phpMtRand, phpRand, print } = require('kolmafia');

const r = phpSeed(1);
const s = phpSeed(2147483647);
print(phpMtRand(r));
print(phpMtRand(s));
print(phpMtRand(r));
print(phpMtRand(s));

r = phpSeed(1);
s = phpSeed(2147483647);
print(phpRand(r));
print(phpRand(s));
print(phpRand(r));
print(phpRand(s));
