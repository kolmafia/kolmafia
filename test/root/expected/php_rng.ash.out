1244335972
844801015
15217923
1915574197
1804289383
1065668062
846930886
2142264300